### PR TITLE
make compatible with fivetran and add readme

### PR DESCRIPTION
**Pull request by eogilvy12**
### Fivetran Compatibility 

* add readme
* create fivetran macros
* update models to pull from macros specific to the ETL specified at the project level
* would love to update the if statements to be a bit more dry

I tested this for a stitch client on snowflake but do not have a client on redshift to test with.

### CODE REVIEW

File: macros/fivetran/base/fivetran_bing_keyword_performance.sql [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        ad_group_id,
+        ad_id,
+        campaign_id,
+        keyword_id,
+        
+        sum(clicks) as clicks,
+        sum(impressions) as impressions,
+        sum(spend) as spend
+    from source
+    group by 1, 2, 3, 4, 5, 6, 7
```
Comment (jthandy) [inline macros/fivetran/base/fivetran_bing_keyword_performance.sql]: `{{dbt_utils.group_by(7)}}`

File: models/base/bing_ad_groups.sql [hunk 2]
```diff
@@ -1 +1,11 @@
-{{ stitch_bing_ad_groups() }}
\ No newline at end of file
+{% if var('etl') == 'stitch' %}
+
+    {{ stitch_bing_ad_groups() }}
+    
+{% endif %}
```
Comment (jthandy) [inline models/base/bing_ad_groups.sql]: don't use two separate if/end blocks...use else :) 

`{% elif var('etl') == 'fivetran' %}`

Comment (jthandy) [reply models/base/bing_ad_groups.sql]: same comment for all of the other models as well.

**Review by jthandy — CHANGES_REQUESTED** [review]
small stuff.

**Comment by eogilvy12** [comment]
@jthandy ready for a final look!

**Review by jthandy — APPROVED** [review]

**Review by ChrisLawliss — APPROVED** [review]

**Pull request merged**